de-lib/src/main/java/org/iplantc/de/fileViewers/client/presenter/FileViewerPresenterImpl.java

```diff
@@ -525,8 +511,6 @@ void composeView(final File file,
 
             if (urls != null && !urls.isEmpty()) {
                 vizViewer.setData(urls);
-            } else if (treeViewer) {
```

Comment:
Wherever this `treeViewer` variable comes from, that line can probably be deleted too